misc/bootstrap-overrides.css

```diff
@@ -5,3 +5,47 @@
 .form-horizontal .col-sm-2.control-label {
```

Comment:
@jeff-phillips-18 thanks!! Should make things a bit clearer to why that file exists if someone sees it in the web inspector